code/game/objects/items.dm

```diff
@@ -269,10 +269,10 @@
 // slot uses the slot_X defines found in setup.dm
 // for items that can be placed in multiple slots
 // note this isn't called during the initial dressing of a player
-/obj/item/proc/equipped(var/mob/user, var/slot)
+/obj/item/proc/equipped(var/mob/user, var/slot, hand_index = 0)
```

Comment:
What I find strange is that you've moved away from the slot system entirely for some reason, instead of adding a slot_hands to signify your hands list.